src/sentry/models/dashboard.py

```diff
@@ -55,6 +69,17 @@ class Meta:
 
     __repr__ = sane_repr("organization", "title")
 
+    @property
+    def favourited_by(self):
+        user_ids = DashboardFavouriteUser.objects.filter(dashboard=self).values_list(
+            "user", flat=True
+        )
+        return user_ids
+
+    @favourited_by.setter
+    def favourited_by(self, user_id):
+        DashboardFavouriteUser.objects.get_or_create(dashboard=self, user=user_id)
```

Comment:
This interface is a little weird, because you return a list of ints, but set an int at a time, like:

```
dashboard = Dashboard()
user_ids = dashboard.favourited_by # This is an empty list
dashboard.favourited_by = 1
user_ids = dashboard.favourited_by # This is now [1]
dashboard.favourited_by = 2
user_ids = dashboard.favourited_by # This is now [1, 2]
```

I'd say have a method like `add_favourite_user` or something similar